test/scip/testdata/field_inheritance.snapshot.rb

```diff
@@ -26,14 +26,16 @@ class C2 < C1
    def get_inherited_ivar
 #      ^^^^^^^^^^^^^^^^^^ definition [..] C2#get_inherited_ivar().
      return @f + @h
-#           ^^ reference [..] C1#`@f`.
-#                ^^ reference [..] C1#`@h`.
+#           ^^ reference [..] C2#`@f`.
+#           relation definition=[..] C1#`@f`.
+#                ^^ reference [..] C2#`@h`.
```

Comment:
This is missing a relation to `C1#@h`

Reply:
So turns out, there is something non-deterministic going on with relationships being emitted sometimes at not at other times...